lib/topology_docker_openswitch/openswitch.py

```diff
@@ -206,16 +202,16 @@ def main():
 
 PROCESS_LOG = """
 #!/bin/bash
-ovs-vsctl list Daemon >> /tmp/logs
-echo "Coredump -->" >> /tmp/logs
-coredumpctl gdb >> /tmp/logs
-echo "All the running processes:" >> /tmp/logs
-ps -aef >> /tmp/logs
+ovs-vsctl list Daemon >> /var/topology/logs
+echo "Coredump -->" >> /var/topology/logs
+coredumpctl gdb >> /var/topology/logs
```

Comment:
Please also add to the logs the output of the stderr (for all the logs), this particular command is leaving output not logged.